endtoend/evaluators/metrics.go

```diff
@@ -19,10 +19,10 @@ import (
 	"google.golang.org/grpc"
 )
 
-const maxMemStatsBytes = 100000000 // 1 MB.
+const maxMemStatsBytes = 2000000000 // 2 GB.
```

Comment:
```suggestion
const maxMemStatsBytes = 2000000000 // 2 GiB.
```

Sorry to be pedantic but 2GB != 2GiB. Gibibytes (GiB) are factors of 10, Gigabytes (GB) are factors of 2.